app/views/components/_govuk_select.html.slim

```diff
@@ -44,7 +44,7 @@ ruby:
           | Clear autocomplete
         svg viewbox="0 0 40 40" class="autocomplete__clear-viewbox"
           path class="autocomplete__clear-icon" d="M 10,10 L 30,30 M 30,10 L 10,30"
-    javascript:
-      window.callAutocompleteWhenReady("#{id}", {showAllValues: "#{local_assigns[:show_all_values]}"});
+    == javascript_tag nonce: true do
+       | window.callAutocompleteWhenReady("#{id}", {showAllValues: "#{local_assigns[:show_all_values]}"});
```

Comment:
`javascript:` by slim just wraps the content between `<script>...</script>` tags.
In order to enable Rails setting the `nonce` value automatically per request, we needed to move away from the slim `javascript:` block and switch to rails `javascript_tag` one.